sdk/web-pubsub/web-pubsub-client/src/models/index.ts

```diff
@@ -102,17 +102,35 @@ export interface LeaveGroupOptions {
 }
 
 /**
- * Send to group operation options
+ * Send to group operation options for fire-and-forget
+ */
+export interface SendToGroupOptionsFireAndForget {
```

Comment:
Would this be SendToGroupFireAndForgetOptions as we typically have the suffix of options

Reply:
@mpodwysocki Updated